### DISCUSSION TITLE
我的项目

### DISCUSSION THREAD

**Discussion by giscus** [zu1k/blog#69] [Comments]
# 我的项目

此页面介绍我做的一些项目和工具

https://zu1k.com/projects/

**Comment by V-know** [comment]
一直在用「关键词机器人」做群管理
今天前来拜读博客

**Comment by koobai** [comment]
你好，现在用DogeCloud COS Action提示Error: fail to upload files to cos: getaddrinfo ENOTFOUND ***.cos.***.myqcloud.com

**Reply by zu1k** [reply]
请到 https://github.com/zu1k/dogecloud-cos-action 反映问题，而不是我博客的评论区

**Comment by XiaoPika** [comment]
真是超级大佬

**Comment by 7lsu** [comment]
为什么越来越偏向Rust了呢
